docusaurus/docs/get-started/set-up-development-environment.mdx

```diff
@@ -15,7 +15,7 @@ import DockerNPM from '@snippets/docker-grafana-version.npm.md';
 import DockerPNPM from '@snippets/docker-grafana-version.pnpm.md';
 import DockerYarn from '@snippets/docker-grafana-version.yarn.md';
 
-The `create-plugin` tool includes a development environment featuring [Docker](https://docs.docker.com/get-docker/). It allows you to start an instance of the Grafana application for plugin developers against which you can code.
+The [`create-plugin` tool](./get-started.mdx#use-plugin-tools-to-develop-your-plugins-faster) includes a development environment featuring [Docker](https://docs.docker.com/get-docker/). It allows you to start an instance of the Grafana application for plugin developers against which you can code.
```

Comment:
<img width="198" alt="Screenshot 2023-10-26 at 19 14 00" src="https://github.com/grafana/plugin-tools/assets/19860021/dbbc4d81-e41b-487f-a6df-0b64c708f84c">

Again this looks a little weird, might want to just have on tool or create-plugin

Reply:
done.